### PR TITLE
Release Toolkits Individually

### PR DESCRIPTION
**Pull request by sdreyer**

### CODE REVIEW

File: .github/workflows/publish-toolkit.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+    - name: Install Python Dependencies
+      run: pip install ./arcade
+
+    - name: Test Toolkit
+      id: Test_Toolkit
+      working-directory: toolkits/${{ steps.set-toolkit.outputs.toolkit }}
+      run: |
+        make install
+        make test
```
Comment (EricGustin) [inline .github/workflows/publish-toolkit.yml:50]: should we add `make check` as well?

Comment (sdreyer) [reply .github/workflows/publish-toolkit.yml:50]: yeah probably

Comment (EricGustin) [reply .github/workflows/publish-toolkit.yml:50]: or maybe that should be done before merging to main?

Comment (sdreyer) [reply .github/workflows/publish-toolkit.yml:51]: we should probably have another workflow that tests and checks them all on commit

**Review by EricGustin — APPROVED** [review]

**Pull request merged**